### DISCUSSION TITLE
start_notify

### DISCUSSION THREAD

**Discussion by ps2229** [hbldh/bleak#819] [Q&A] [answered]
I just started using bleak for android can anyone tell me what does start_notify do exactly , was able to understand all the other functions but could not understand start_notify what does it do and why is it used? Thank you for the help.

**Comment by dlech** [comment, marked as answer]
It registers a callback that will be called when notifications are received on the host system`self.__gatt.setCharacteristicNotification(characteristic.obj, True)` and it writes to the characteristic descriptor on the remote device to tell the remote device to enable notifications ` self.write_gatt_descriptor(characteristic.notification_descriptor, defs.BluetoothGattDescriptor.ENABLE_NOTIFICATION_VALUE)`

**Comment by ps2229** [comment]
can you explain with an example that would be better as I was trying out examples for start_notify in android but facing alot of issues.
